modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc

```diff
@@ -7,9 +7,27 @@ This page lists new features added in Redpanda Cloud.
 
 == May 2024
 
+=== Cloud API: beta
+
+The Cloud API allows you to programmatically manage clusters and resources in your Redpanda Cloud organization. For more information, see the xref:deploy:deployment-option/cloud/api/cloud-api-quickstart.adoc[Cloud API Quickstart], the xref:deploy:deployment-option/cloud/api/cloud-api-overview.adoc[Cloud API Overview], and the full xref:api:ROOT:cloud-api.adoc[Cloud API reference].
+
+=== Redpanda Connect 
+
+xref:redpanda-connect:ROOT:about.adoc[Redpanda Connect] includes over 200 connectors, with a large collection of sources, sinks, and processors. The interface provides a lightweight and easy way to set up, debug, and manage connectors. 
+
+=== mTLS authentication for Kafka API clients
+
+mTLS authentication is now available for Kafka API clients. You can xref:deploy:deployment-option/cloud/security/cloud-authentication.adoc#mtls[enable mTLS] for your cluster using the Cloud API.
+
+=== Manage private connectivity in the UI
+
+You can now manage GCP Private Service Connect and AWS PrivateLink connections to your BYOC or dedicated cluster on the *Cluster settings* page in Redpanda Cloud. See the steps for xref:deploy:deployment-option/cloud/configure-privatelink-in-cloud-ui.adoc[PrivateLink] and xref:deploy:deployment-option/cloud/configure-private-service-connect-in-cloud-ui.adoc[Private Service Connect].
+
+To add private connectivity as part of cluster creation, use the Cloud API. 
```

Comment:
> To add private connectivity as part of cluster creation, use the Cloud API. 

Not sure what this line means ?

Reply:
@bpraseed is it correct that in the UI, you can add PrivateLink/PSC to an existing cluster, but if you want to add them as part of the cluster creation step, you need to do it with the Cloud API? Or, even if correct, do we not want to make that distinction?